packages/babel-plugin/src/class-names/index.ts

```diff
@@ -132,15 +132,12 @@ export const visitClassNamesPath = (path: NodePath<t.JSXElement>, meta: Metadata
       }
 
       const builtCss = buildCss(styles, meta);
-      const { sheets, classNames } = transformCss(
-        builtCss.css.map((x) => getItemCss(x)).join(''),
-        meta.state.opts
-      );
+      const { sheets, classNames } = transformCssItems(builtCss.css, meta);
```

Comment:
The change in this file is to make <ClassNames> handle conditional CSS rules.
i.e. 
```
<ClassNames>
          {({ css }) => (<div className={css({ ...props.isPrimary && { color: 'blue' }})}>hello, world!</div>)}
</ClassNames>
```